main.go

```diff
@@ -27,6 +28,12 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
+func PrintUsage() {
```

Comment:
The intention here was to have an exported function that was callable from main.go per your [colleague's request](https://goto.google.com/tpfsc).

Reply:
Hmm I'm not sure we need to actually call a function. We can just execute `keep-sorted --help` to get the same output

Reply:
@JeffFaer the existing (automatic) help text seems a bit wrong. This CL notably documents positional args. The current `--help` text seems to be:

```
$ ./main --help
Usage of ./main:
      --color string              Whether to color debug output. One of "always", "never", or "auto" (default "auto")
      --default-options options   The options keep-sorted will use to sort. Per-block overrides apply on top of these options. Note: list options like prefix_order are not merged with per-block overrides. They are completely overridden. (default allow_yaml_lists=yes case=yes group=yes remove_duplicates=yes sticky_comments=yes)
      --lines line_ranges         Line ranges of the form "start:end". Only processes keep-sorted blocks that overlap with the given line ranges. Can only be used when fixing a single file. This flag can either be a comma-separated list of line ranges, or it can be specified multiple times on the command line to specify multiple line ranges. (default [])
      --mode mode                 Determines what mode to run this tool in. One of ["fix" "lint"] (default fix)
  -v, --verbose count             Log more verbosely
pflag: help requested
```

And the `-` behavior doesn't seem documented at all. The wrapper we have is even worse. I didn't check how this CL exactly will act there (I think our wrapper doesn't play nice with `os.Args[0]`), but it should at least improve things by documenting the positional args.

As for an exported method, I don't follow the ask for that. We can't override it even if it were exported (not that it's mutable either).

Reply:
> We can just execute keep-sorted --help to get the same output

The flags for the external `main.go` differ from the internal `main.go`, thus the Usage functions must differ.

@JeffFaer please explicitly state how you would like this structured

Reply:
I have inlined this function as the request to share the logic has been dropped.

goto.google.com/drcao